### PR TITLE
Vitamin buff and removal from plants + frying and sugar nerf

### PR DESCRIPTION
**Pull request by Marmio64**
### Intent of your Pull Request
To makes botany plants and chemistry slightly less good at replacing the chef's job, and to make frying all your food as a chef not be the clear way of feeding the station

also the intent is to massively improve RP, like, merging this will QUADRUPLE the amount of rp in the game, not even joking.

### Why is this change good for the game?
Encourages making crafted food as a chef, since other food sources aren't as good now and vitamins heal more, along with standard nutriments healing less.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Removes all vitamins from ALL plants. Nutriment amount is kept.
Makes vitamins heal 50% more and nutriments heal 50% less.
Cooking oil and sugar is much less nutritious.

### What should players be aware of when it comes to the changes your PR is implementing?
that fried shit and sugar isn't so good at feeding you now, eat something healthy you fucks

### What general grouping does this PR fall under? 
Chef, botany, and chem changes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
vitamins heal 1.5 brute and burn and nutriments only heal 0.5 brute now.

# Changelog

:cl:  
tweak: sugar and cooking oil less nutritious, vitamins heal more, nutriment heals less, no more vitamins in plants.  
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Okay but vegetables are good for you tho

**Comment by Marmio64** [comment]
> Okay but vegetables are good for you tho

yes but in real life they don't take away the job of every chef in the world, so we have to make some compromises

**Comment by boodaliboo** [comment]
> > Okay but vegetables are good for you tho
> 
> yes but in real life they don't take away the job of every chef in the world, so we have to make some compromises

but cooks dont heal people irl

**Comment by Marmio64** [comment]
> > > Okay but vegetables are good for you tho
> > 
> > 
> > yes but in real life they don't take away the job of every chef in the world, so we have to make some compromises
> 
> but cooks dont heal people irl

they heal people emotionally, with good food

**Comment by Tipy1802** [comment]
> > > > Okay but vegetables are good for you tho
> > > 
> > > 
> > > yes but in real life they don't take away the job of every chef in the world, so we have to make some compromises
> > 
> > 
> > but cooks dont heal people irl
> 
> they heal people emotionally, with good food

Thats called a mood buff not healing :P

**Comment by Marmio64** [comment]
> > > > > Okay but vegetables are good for you tho
> > > > 
> > > > 
> > > > yes but in real life they don't take away the job of every chef in the world, so we have to make some compromises
> > > 
> > > 
> > > but cooks dont heal people irl
> > 
> > 
> > they heal people emotionally, with good food
> 
> Thats called a mood buff not healing :P

at most, 5% of the people who play yogstation uses mood. The incredibly short lived mood buff from food does not matter to almost everyone playing because mood isn't mandatory, and to turn it on is to put yourself at a disadvantage

**Comment by SomeguyManperson** [comment]
nutriment probably doesn't need less healing since it's super secondary

**Review by jcatjmeow — APPROVED** [review]

**Comment by Cartlord** [comment]
You seem to be failing to consider the times when there is no Chef, or Chef is dead/antag
Way I see it, Botany + Chems SHOULD be able to do Chef's job - that's two departments working together (to an extent) to cover for one.

**Comment by Marmio64** [comment]
> You seem to be failing to consider the times when there is no Chef, or Chef is dead/antag
> Way I see it, Botany + Chems SHOULD be able to do Chef's job - that's two departments working together (to an extent) to cover for one.

they legit still can do his job, botany will still be able to mass produce melons for food and sugar will still fill you up, so your argument means nothing.

**Comment by Sniblet** [comment]
miners have to eat more often now :weary:

**Comment by Sniblet** [comment]
Oh my god this is agony please make plants useful as food again

**Comment by Marmio64** [comment]
> Oh my god this is agony please make plants useful as food again

are you saying 20-40 nutriment melons that botany can mass produce in order to feed the whole station for dozens of hours within 15 minutes isn't useful as food? Those same melons can also be put biogenerator to mass produce flour for bread. Even if you're shit at botany, you can still make large amounts of 10 potency potatoes. Also all lavaland plants were unaffected by vitamin removal so I don't know what you're talking about when you say miners need to eat more often.

**Comment by Sniblet** [comment]
I suppose I misunderstood you when you said "ALL plants," it looks like Lavaland plants weren't changed.

20-40 nutriment melons might sound like a lot, but nutriment is not good for nutrition. Next to vitamins it's really, really bad, and you have to go back to hydroponics every 5-10 minutes if you're playing vegetarian, and you get fat much more easily.

Why would you even do this? I still don't understand. We know it doesn't make sense logically, but how does it make sense in balance either?
Is it about healing? If you're going to botany for healing, you're going there for their omnizine, not for their vitamins.
Is it to buff cook as a nutrition source? Why do you have to do that by bringing botany down? Suppose the cook is dead or an idiot or never signed up - he's not only missed, he's irreplaceable, there's no other source of vitamin, so all there is is effectively junk food unless people break into the kitchen.
Yes, people _can_ live that way, but why do they need to? Just make the cook's food stronger, nobody will mind.
